docs/fold.md

```diff
@@ -0,0 +1,82 @@
+## AIR: Fold instruction
+Here is the `fold` syntax.
+
+`(fold <iterable> <iterator> <instruction> [<last_instruction>])`
+
```

Comment:
Let add here that `Iterable` can be a stream, map, canonicalized stream, canonicalized map or scalar array.

Reply:
Fixed